packages/abi-typegen/src/utils/getTypeDeclaration.ts

```diff
@@ -0,0 +1,31 @@
+import { TargetEnum } from '../types/enums/TargetEnum';
+import type { IType } from '../types/interfaces/IType';
+import type { JsonAbiArgument } from '../types/interfaces/JsonAbi';
+
+import { findType } from './findType';
+import { parseTypeArguments } from './parseTypeArguments';
+
+export function resolveInputLabel(
```

Comment:
We could add a unit test around this functionality?

Reply:
There is no need, this is just a helper function I extracted from the `Function` class so that it could be reused in the configurable. The tests in `/src/templates/` verifying that functions and configurables are generated properly are enough. A lot of work I had to do in #2826 was around making these utils tests happy while higher-level tests testing the generated code were already green.